apps/feedahelp/src/pages/_app.tsx

```diff
@@ -11,24 +11,34 @@ import { type AppType } from "next/app";
 import { trpc } from "../utils/trpc";
 import { appWithTranslation } from "next-i18next";
 import "~/styles/globals.css";
+import { PayPalScriptProvider } from "@paypal/react-paypal-js";
 
 const MyApp: AppType<{ session: Session | null }> = ({
   Component,
   pageProps: { session, ...pageProps },
 }) => {
   return (
+
     <FormatImageUrlProvider value={formatImageUrl}>
       <SessionProvider session={session}>
         <LoadingContextProvider>
           <MuiThemeProvider theme={baseTheme}>
             <CssBaseline />
             <CmsContextProvider>
+            <PayPalScriptProvider
```

Comment:
This provider is only for paypal? what other providers? can we add multiple options?